utils/markdown.js

```diff
@@ -162,12 +163,20 @@ function buildTree(lines) {
 
     // Remaining items respect indent-based nesting.
     const [, indent, content] = /** @type {string[]} */ (line.match('^([ ]*)(.*)'));
+    const [codeLang, title] = (() => {
+      const lang = content.substring(3);
+      const match = lang.match(/ title="(.+)"/);
```

Comment:
I think we should support any ordering between `title` and everything else, so perhaps split by space and find the title?

Reply:
discussed offline, leaving it out for now.